codalab/worker/worker_run_state.py

```diff
@@ -290,8 +289,9 @@ def mount_dependency(dependency, shared_file_system):
         if not dependencies_ready:
             status_message = status_messages.pop()
             if status_messages:
-                status_message += "(and downloading %d other dependencies and docker images)" % len(
-                    status_messages
+                status_message += (
+                    "\n(and downloading %d other dependencies and docker images)"
```

Comment:
I don't see a newline in the latest screenshot you sent. I'm guessing perhaps the newline just doesn't show up in the HTML; in that case, let's just remove the newline.